create_infotexts.py

```diff
@@ -195,6 +195,8 @@ def main():
         img = CypernImage()
         img.generate_list_of_stripped_keywords(metadata[fotonr]["Nyckelord"])
         img.create_commons_filename(metadata[fotonr])
+        img.add_cat_archaeological_exhibition_based_on_description(metadata[fotonr]["Beskrivning"])
```

Comment:
A) Done
B) I disagree. "Each function should do one thing and one thing only" and to understand special cases flow afterwards I think the naming of functions is more precise that way. OK?